### PR TITLE
Docs/add brazilian portuguese translation

### PR DESCRIPTION
**Pull request by pvgomes**
## What Changed
Added and completed the new pt-BR documentation set, including missing command and example translations, and updated language navigation in the main README.

### Scope delivered:

- Added docs/pt-BR with 47 translated files in total 
- Included core pt-BR docs: README, CONTRIBUTING, and TERMINOLOGY
- Completed translations for commands and examples sections
- Added Portuguese (Brasil) to language switchers in the main README
- Updated language count in the main README from 5 to 6 supported languages

**Breakdown:**

- 12 agent docs
- 18 command docs
- 6 example docs
- 8 rule docs
- 3 core docs

## Why This Change
Portuguese is a high-impact language for documentation accessibility, not a niche addition.

- Portuguese has roughly 267 million total speakers globally (about 250 million L1 + 17 million L2), making it one of the most spoken languages in the world.
- It is an official language across 9 countries and has institutional use across multiple regions through the Lusophone bloc (CPLP), so the audience is geographically broad.
- Brazil alone has about 205.3 million people and is the largest economy/population in Latin America, which makes Portuguese support strategically valuable for adoption and community growth.
- Completing pt-BR docs reduces onboarding friction for a very large developer base and helps keep parity with other translated documentation sets

## Testing Done
- [x] Manual testing completed
- [ ] Automated tests pass locally (`node tests/run-all.js`)
- [ ] Edge cases considered and tested

## Type of Change
- [ ] `fix:` Bug fix
- [ ] `feat:` New feature
- [ ] `refactor:` Code refactoring
- [x] `docs:` Documentation
- [ ] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist
- [x] No secrets or API keys committed (ghp_, sk-, AKIA, xoxb, xoxp patterns checked)
- [ ] JSON files validate cleanly
- [ ] Shell scripts pass shellcheck (if applicable)
- [ ] Pre-commit hooks pass locally (if configured)
- [x] No sensitive data exposed in logs or output
- [x] Follows conventional commits format

## Documentation
- [x] Updated relevant documentation
- [ ] Added comments for complex logic
- [ ] README updated (if needed)

---
## Summary by cubic
Added full Brazilian Portuguese (pt-BR) docs and updated the README language switcher to include it. The site now supports 6 languages.

- **New Features**
  - Added `docs/pt-BR` with core docs and all agents, commands, examples, and rules (47 files).
  - Refined key guides: another pass on `/eval` (criteria/reporting), plus `/go-review`, `go-build-resolver`, `security-reviewer`, `refactor-cleaner`, and improved `/orchestrate` and `/plan` workflows.
  - Updated `README.md` to add Português (Brasil) and bump language count from 5 to 6.

<sup>Written for commit c3769b5c1348606afda186be655c6a2628b2a84a. Summary will update on new commits.</sup>

## Summary by CodeRabbit

* **New Features**
  * Added Português (Brasil) to the language selector and site metadata; updated featured languages count from 5 to 6.

* **Documentation**
  * Major Portuguese (pt-BR) docs release: contribution guide, glossary, many agent/command guides, examples and blueprints for multiple stacks, and comprehensive rules on security, testing, coding style, git workflow, hooks, performance, and tooling.

### CODE REVIEW

File: docs/pt-BR/commands/orchestrate.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+Approvals:
+- pending user approvals
+- blocked steps awaiting confirmation
+
+Telemetry:
+- last activity timestamp or idle signal
+- estimated token or cost drift
+- policy events raised by hooks or reviewers
+```
```
[the remark this answers is not in the document]
Comment (pvgomes) [reply docs/pt-BR/commands/orchestrate.md:205]: this is ok to not be

**Review by affaan-m — APPROVED** [review]
Comprehensive Brazilian Portuguese translation matching existing doc structure. Clean docs-only contribution. Thanks @pvgomes!
